src/macro_util.rs

```diff
@@ -61,6 +65,134 @@ impl<T, U> MaxAlignsOf<T, U> {
     }
 }
 
+const _64K: usize = 1 << 16;
+
+// remove this `cfg` when `size_of_val_raw` is stabilized
+#[cfg(__INTERNAL_USE_ONLY_NIGHLTY_FEATURES_IN_TESTS)]
+#[repr(C, align(65536))]
+struct Aligned64kAllocation([u8; _64K]);
+
+/// A pointer to an aligned allocation of size 2^16.
+///
+/// # Safety
+///
+/// `ALIGNED_64K_ALLOCATION` is guaranteed to point to the entirety of an
+/// allocation with size and alignment 2^16, and to have valid provenance.
+// remove this `cfg` when `size_of_val_raw` is stabilized
```

Comment:
```suggestion
// TODO(#29), TODO(https://github.com/rust-lang/rust/issues/69835): Remove this `cfg` when
// `size_of_val_raw` is stabilized.
```

(but reflowed to 80 columns, which I assume my suggestion isn't)

Same elsewhere in this file.